src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Button from '../../../../common/Button'
+import { StyledCourseCard } from '../../../../common/CommonHTML'
+import { mockedAuthorsList } from '../../../../constants'
+import getDuration from '../../../../helpers/getCourseDuration'
+
+export default function CourseCard(props) {
```

Comment:
Defined a component with `props` may not be a good idea.
We could define a component with clean props, like:
```js
export default function CourseCard({
  title,
  description,
  creationDate,
  duration,
  authors
})
```
After that, we could easily read and modify after that.
Now, we found another issue with `creationDate`